<ui version="4.0" >
 <class>ColorEditDialogClass</class>
 <widget class="QDialog" name="ColorEditDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ColorEditDialog</string>
  </property>
  <widget class="ColorWheelWidget" native="1" name="colorWheel" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>10</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorWidget" native="1" name="colorWidget" >
   <property name="geometry" >
    <rect>
     <x>6</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorVolumeWidget" native="1" name="colorVolumeLight" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorVolumeWidget" native="1" name="colorVolumeDark" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeButton" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>0</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>x</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pickButton" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>160</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>ColorWheelWidget</class>
   <extends>QWidget</extends>
   <header>colorwheelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>colorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorVolumeWidget</class>
   <extends>QWidget</extends>
   <header>colorvolumewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
